docs/configuration/adding_synthesizers.md

```diff
@@ -133,6 +133,18 @@ SynthDef(\elegiac, {
 }).add;
 ```
 
+### Using custom parameters
+
+If you want to create any custom parameter for your `SynthDef`, it also has to be referenced in **Tidal**.
+To do this, you have to create a parameter in **Tidal** with the same name the argument has in **SuperCollider**.
+For example, if the arguments in **SuperCollider** were `| harm, pit, model |`, you should add this to your [Tidal Boot File](https://tidalcycles.org/docs/configuration/boot-tidal/):
+
+```haskell
+let harm = pF "harm"
+let pit = pF "pit"
+let model = pI "model" 
+```
+
```

Comment:
You could add a note that in SuperDirt, you can generate the parameter list for a list of `SynthDef` names:
```supercollider
SuperDirt.postTidalParameters([\imp, \default])
```

Reply:
Do the synths need to be in a specific path?
I am running
````supercollider
SuperDirt.postTidalParameters([\supersaw])
````
and getting
`WARNING: no Synth Description with this name found: supersaw`

Reply:
I did not get that error running the postTidalParameters on supersaw. The supersaw SynthDef is in:
SuperCollider/downloaded-quarks/SuperDirt/library/default-synths-extra.scd

The postTidalParameters command is useful. I'll add it after this is merged.